modules/core/04-channel/types/msgs_test.go

```diff
@@ -695,6 +695,105 @@ func (suite *TypesTestSuite) TestMsgChannelUpgradeAckValidateBasic() {
 	}
 }
 
+func (suite *TypesTestSuite) TestMsgChannelUpgradeConfirmValidateBasic() {
+	var msg *types.MsgChannelUpgradeConfirm
+
+	testCases := []struct {
+		name     string
+		malleate func()
+		expPass  bool
```

Comment:
noting that we should switch these to check for errors. given that all current tests don't do this, I'll open an issue to keep track of it and punt it for later.

Reply:
Agreed! Just kept the existing format to conform for now 👍